components/InfoCard.tsx

```diff
@@ -0,0 +1,21 @@
+import * as React from 'react';
+import styles from '../styles/InfoCard.module.css';
+
+interface InfoCardProps {
+    Title: string;
+    Image: string;
+    Body: string;
+}
+
+const InfoCard: React.FC<InfoCardProps> = (props: InfoCardProps): React.ReactElement => {
+  const { Title, Image, Body } = props;
+  return (
+    <div>
+      {Image && <img src={Image} alt="InfoCard" className={styles.ICImage} />}
```

Comment:
even though this is a somewhat small site, using [`next/image`](https://nextjs.org/docs/api-reference/next/image) is preferred here for images! helps with _optimizations_.

additionally, `width` and `height` are solid attributes to have. i see they're in the styles, but we've had some problems with rendering these in the past 🥴 

and the `alt` text is fine for now, but this will always be passed from contentful. once that is setup, this can be easily plugged in!